### ISSUE TITLE
What is authorName and where is it coming from?

### ISSUE THREAD

**Issue by Toflar** [contao/news-bundle#9]
I must be blind...can anybody explain me where this line takes the author name from? https://github.com/contao/news-bundle/blob/46256caff85ef2cd0b72b2235ed693a5b7876f53/src/Resources/contao/classes/News.php#L184

I cannot find the logic behind this.

**Comment by fritzmg** [comment]
Looks indeed like a relic from Contao 2. As a matter of fact, no author information is generated in the RSS feeds. Tested on demo.contao.org:

RSS 2.0 entry:
```xml
<item>
  <title>Contao is popular</title>
  <description><![CDATA[<p>In a recent security level comparison Contao excelled as one of the most secure Open Source CMS's.</p>]]></description>
  <link>https://demo.contao.org/en/news-detail/contao-is-popular.html</link>
  <pubDate>Mon, 17 Feb 2014 14:27:00 +0100</pubDate>
  <guid>https://demo.contao.org/en/news-detail/contao-is-popular.html</guid>
</item>
```
Atom entry:
```xml
<entry>
  <title>Contao is popular</title>
  <content type="xhtml"><div xmlns="http://www.w3.org/1999/xhtml"><p>In a recent security level comparison Contao excelled as one of the most secure Open Source CMS's.</p></div></content>
  <link rel="alternate" href="https://demo.contao.org/en/news-detail/contao-is-popular.html" />
  <updated>2014-02-17T14:27:00+01:00</updated>
  <id>https://demo.contao.org/en/news-detail/contao-is-popular.html</id>
  <author><name></name></author>
</entry>
```
With the Atom entry you can see that an `<author>` node is in fact added to the `<entry>`, however it is empty.

**Comment by leofeyer** [comment]
The author name should appear in the Atom feed.

**Comment by fritzmg** [comment]
It works in neither feed. While the Atom feed does add an `<author>` entry, it will be empty since `authorName` is a non-existant variable of the model. It should be `->getRelated('author')->name` instead for example (with null check).

**Comment by leofeyer** [comment]
Fixed in 9f46a7d3d6a767e3374cfe24379fda1762531579.
